website/integrations/services/owncloud/index.md

```diff
@@ -0,0 +1,223 @@
+---
+title: Integrate with ownCloud
+sidebar_label: ownCloud
+---
+
+<span class="badge badge--secondary">Support level: Community</span>
+
+## What is ownCloud
+
+> ownCloud is a free and open-source software project for content collaboration and sharing and syncing of files.
+>
+> -- https://owncloud.com
+
+:::note
+This guide focuses on ownCloud installations that are deployed using Docker. If you have deployed it using a different mechanism, there may be some differences in the process.
```

Comment:
```suggestion
This guide focuses on ownCloud installations that are deployed using Docker. If you deployed ownCloud using a different mechanism, there might be some differences in the process.
```

Reply:
just in case you're curious about the edits, in technical writing you try to avoid using "it" and we use "might" to indicate chance... "may" is used to indicate permission. ;-)